cc.newspring.Apollos/Rest/Controllers/GiveController.cs

```diff
@@ -206,504 +274,571 @@ public HttpResponseMessage ScheduleGiving( [FromBody]ScheduleParameters schedule
         [System.Web.Http.Route( "api/Give" )]
         public HttpResponseMessage Give( [FromBody]GiveParameters giveParameters )
         {
-            var errorResponse = ValidateGiveParams( giveParameters );
+            var rockContext = new RockContext();
 
-            if ( errorResponse != null )
+            try

```

Comment:
Correct.  In order to get errors and send them back to the REST client user, I needed to wrap with a try/catch block.